CHANGELOG.rst

```diff
@@ -50,6 +65,7 @@ Changed
 - Do not retrain the entire Core model if only the ``templates`` section of the domain
   is changed.
 - Upgraded ``jsonschema`` version.
+- Updated ``/status`` endpoint to show correct path to model archive
```

Comment:
omg, after all that changelog things we still added it at the wrong place 😑 can you please move that to 1.5.1?

Reply:
Hahaha sure